app/javascript/css/calculator.css.scss

```
@@ -141,6 +141,10 @@
   }
 }
 
+.close-img{
+  z-index: 1;
+}
```

Use something like this
```suggestion
.z-1 {
  z-index: 1;
}
```

---

done